sample/package.json

```diff
@@ -20,7 +20,7 @@
     "ecosystem:cordova"
   ],
   "devDependencies": {
-    "cordova-android": "^12.0.1",
```

Comment:
These changes seem to be not relevant for the JS upgrade. Likely leaking from the Android SDK bump.